item-review-viewer/src/main/java/org/smarterbalanced/itemreviewviewer/web/services/GitLabService.java

```diff
@@ -665,19 +666,15 @@ private void _downloadStim(String namespace, String bankKey, String id) {
     }
 
     @Override
```

Comment:
Weren't we planning to have this function handle multiple items? Other wise we need to make a request per item on the front end which will be slow. I'd prefer to pass it the whole list of items and have it process them all at once.

Reply:
This function is used in the controller side which iteratively calls it.
So the logic of calling multiple items in RenderItemController.java file line: 258